test/lib/NonLinearDutchDecayLib.t.sol

```diff
@@ -11,17 +11,18 @@ import {Uint16Array, toUint256} from "../../src/types/Uint16Array.sol";
 import {Math} from "openzeppelin-contracts/utils/math/Math.sol";
 import {ArrayBuilder} from "../util/ArrayBuilder.sol";
 import {CurveBuilder} from "../util/CurveBuilder.sol";
+import {MockERC20} from "../util/mock/MockERC20.sol";
+import {OutputToken, InputToken} from "../../src/base/ReactorStructs.sol";
 
 /// @notice mock contract to test NonlinearDutchDecayLib functionality
 contract MockNonlinearDutchDecayLibContract {
     function decayInput(
         NonlinearDutchDecay memory curve,
         uint256 startAmount,
         uint256 decayStartBlock,
-        uint256 minAmount,
         uint256 maxAmount
     ) public view {
-        NonlinearDutchDecayLib.decayInput(curve, startAmount, decayStartBlock, minAmount, maxAmount);
+        decayInput(curve, startAmount, decayStartBlock, maxAmount);
     }
```

Comment:
Isn't this recursive?

Reply:
haha we actually don't need this altogether.. thanks for flagging!